src/cgame/cg_consolecmds.cpp

```diff
@@ -1556,14 +1530,28 @@ qboolean CG_ConsoleCommand(void)
 
 	cmd = CG_Argv(0);
 
-	for (i = 0 ; i < static_cast<int>(sizeof(commands) / sizeof(commands[0])) ; i++)
+	// check for commands allowed anytime
+	for (i = 0 ; i < static_cast<int>(sizeof(anyTimeCommands) / sizeof(anyTimeCommands[0])) ; i++)
 	{
-		if (!Q_stricmp(cmd, commands[i].cmd))
+		if (!Q_stricmp(cmd, anyTimeCommands[i].cmd))
 		{
-			commands[i].function();
+			anyTimeCommands[i].function();
 			return qtrue;
 		}
 	}
+	
+	if (!cg.demoPlayback)
```

Comment:
Rather than checking if we're **not** playing demo, check if we **are** and error out?

```cpp
if (cg.demoPlayback)
{
    CG_Printf( <command> not allowed during demo playback);
    return;
}

for (i = 0 ; i < static_cast<int>(sizeof(noDemoCommands) / sizeof(noDemoCommands[0])) ; i++)
{
    ...
}
```

Reply:
We cannot error out early because in functions further down there is separate parsing for other commands not handled by the two new arrays